<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressBarDialog</class>
 <widget class="QDialog" name="ProgressBarDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Progress Bar</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/static/default/icon/22x22/moonstone.png</normaloff>:/static/default/icon/22x22/moonstone.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="loadingText">
     <property name="text">
      <string>Loading...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="text">
        <string>Stop</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/static/default/icon/22x22/process-stop.png</normaloff>:/static/default/icon/22x22/process-stop.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/static/default/icon/22x22/dialog-cancel.png</normaloff>:/static/default/icon/22x22/dialog-cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
